web/crux/src/config/config.config.ts

```diff
@@ -0,0 +1,6 @@
+const configModuleConfig = {
```

Comment:
The file name would be nicer just simply as `app.config.ts` IMHO.

Reply:
Yeah, good point. I'll rename the object too.